src/Internal.AspNetCore.Sdk/build/Common.props

```diff
@@ -12,12 +12,14 @@ Usage: this should be imported once via NuGet at the top of the file.
   <PropertyGroup>
     <Authors>Microsoft</Authors>
     <Company>Microsoft Corporation.</Company>
-    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
+    <Copyright>Copyright © Microsoft Corporation</Copyright>
     <IncludeSymbols>true</IncludeSymbols>
     <NeutralLanguage>en-US</NeutralLanguage>
     <NoPackageAnalysis>true</NoPackageAnalysis>
-    <ProjectUrl>https://asp.net</ProjectUrl>
-    <RequireLicenseAcceptance>true</RequireLicenseAcceptance>
+    <PackageLicenseUrl>https://raw.githubusercontent.com/aspnet/Home/2.0.0/LICENSE.txt</PackageLicenseUrl>
```

Comment:
I did not create a 2.0.0 tag in Home repo yet and wanted to wait when we actually create tags for all our repos.